tests/unit/amo/components/TestAddon.js

```diff
@@ -746,6 +711,83 @@ describe(__filename, () => {
     expect(rootNode.querySelector('.AddonDescription')).toEqual(null);
   });
 
+  it("does not display a lightweight theme's show more card", () => {
+    const root = shallowRender({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        summary: 'my theme is very cool',
+      }),
+    });
+
+    expect(root.find(ShowMoreCard)).toHaveLength(0);
```

Comment:
I think we should find `.AddonDescription`, it is more explicit IMO.

Reply:
oh ok, I see...